dist/t/spec/features/0060_build_result_spec.rb

```diff
@@ -0,0 +1,73 @@
+require "spec_helper"
+#for getting spec file
+require 'tmpdir'
+require "net/https"
+require "uri"
+
+RSpec.describe "Checking Build Results" do
+
+  it "should be able to Overview Build Results" do
+    visit("/project/show/home:Admin")
+    click_link('Overview')
+    expect(page).to have_content("Build Results")
+  end
+
+  it "should be able to check Build Results and see succeeded package built" do
+    sleep(10)
+    while page.has_content?("State needs recalculation") do
+      sleep(3)
+      find("img[alt='Reload']").click
```

Comment:
I didn't had good experience with this reload, especially on selenium.
Maybe direct call to JS reload would work better: `page.evaluate_script("window.location.reload()")`